pkg/apis/pravega/v1alpha1/bookkeeper.go

```diff
@@ -126,6 +133,27 @@ func (s *BookkeeperSpec) withDefaults() (changed bool) {
 	return changed
 }
 
+// BookkeeperImageSpec defines the fields needed for a BookKeeper Docker image
+type BookkeeperImageSpec struct {
+	ImageSpec
+}
+
+func (s *BookkeeperImageSpec) withDefaults(c *ClusterSpec) (changed bool) {
```

Comment:
Same here, argument seems not in use.

Reply:
Right. I've set the tag to `.spec.Version`.